Globally installed sw-cli throws unhelpful errors

**Library Affected**:
sw-cli

**Browser & Platform**:
NodeJS v7.1.0
NPM 3.10.9
MacOS 10.12.3

**Issue or Feature Request Description**:
I ran `npm install -g sw-cli` and then ran `sw-cli generate-sw`

Observed behavior:
```
$ sw-cli
module.js:474
    throw err;
    ^

Error: Cannot find module 'glob'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.Module._load (module.js:420:25)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/stephenfluin/.nvm/versions/node/v7.1.0/lib/node_modules/sw-cli/node_modules/sw-build/build/lib/utils/get-file-details.js:1:76)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
```

Expected behavior:
Either it to work, or to throw a helpful error message about not installing sw-cli globally.

--- a/package.json
+++ b/package.json
@@ -37,6 +37,7 @@
     "chromedriver": "^2.26.1",
     "clear-require": "^2.0.0",
     "cookie-parser": "^1.4.3",
+    "depcheck": "^0.6.7",
     "error-stack-parser": "^2.0.0",
     "eslint-config-google": "^0.7.0",
     "express": "^4.14.0",

--- a/packages/sw-build/test/dependency-check.js
+++ b/packages/sw-build/test/dependency-check.js
@@ -1,0 +1,36 @@
+const path = require('path');
+const depcheck = require('depcheck');
+
+describe('Test Dependencies', function() {
+  it('should have required dependencies', function() {
+    return new Promise((resolve, reject) => {
+      depcheck(path.join(__dirname, '..'), {
+      ignoreDirs: [
+        'test',
+        'build',
+        'demo',
+      ],
+      ignoreMatches: [
+        'sw-lib',
+      ],
+    }, (unusedDeps) => {
+      if (unusedDeps.dependencies.length > 0) {
+        console.log(unusedDeps.dependencies);
+        return reject(new Error('Unused dependencies defined in package.json'));
+      }
+
+      if (unusedDeps.devDependencies.length > 0) {
+        console.log(unusedDeps.dependencies);
+        return reject(new Error('Unused devDependencies defined in package.json'));
+      }
+
+      if (Object.keys(unusedDeps.missing).length > 0) {
+        console.log(unusedDeps.missing);
+        return reject(new Error('Dependencies missingfrom package.json'));
+      }
+
+      resolve();
+    });
+    });
+  });
+});

--- a/packages/sw-cli/package.json
+++ b/packages/sw-cli/package.json
@@ -32,7 +32,6 @@
     "chalk": "^1.1.3",
     "inquirer": "^2.0.0",
     "minimist": "^1.2.0",
-    "mkdirp": "^0.5.1",
     "sw-build": "^0.0.17",
     "update-notifier": "^1.0.3"
   },

--- a/packages/sw-cli/test/dependency-check.js
+++ b/packages/sw-cli/test/dependency-check.js
@@ -1,0 +1,35 @@
+const path = require('path');
+const depcheck = require('depcheck');
+
+describe('Test Dependencies', function() {
+  it('should have required dependencies', function() {
+    return new Promise((resolve, reject) => {
+      depcheck(path.join(__dirname, '..'), {
+      ignoreDirs: [
+        'test',
+        'build',
+        'demo',
+      ],
+      ignoreMatches: [
+      ],
+    }, (unusedDeps) => {
+      if (unusedDeps.dependencies.length > 0) {
+        console.log(unusedDeps.dependencies);
+        return reject(new Error('Unused dependencies defined in package.json'));
+      }
+
+      if (unusedDeps.devDependencies.length > 0) {
+        console.log(unusedDeps.dependencies);
+        return reject(new Error('Unused devDependencies defined in package.json'));
+      }
+
+      if (Object.keys(unusedDeps.missing).length > 0) {
+        console.log(unusedDeps.missing);
+        return reject(new Error('Dependencies missingfrom package.json'));
+      }
+
+      resolve();
+    });
+    });
+  });
+});

